### ISSUE TITLE
docs: cover timeout scaling in timeout FAQ

### ISSUE THREAD

**Issue by mattklein123** [envoyproxy/envoy#14775]
We should cover the new scaled timeout capability in the timeout FAQ: https://www.envoyproxy.io/docs/envoy/latest/faq/configuration/timeouts

We should probably also cover it in the edge best practices guide:
https://www.envoyproxy.io/docs/envoy/latest/configuration/best_practices/edge#best-practices-edge

**Comment by mattklein123** [comment]
@akonradi would you mind covering this when you feel the scaling work is in a good spot? (Or just do it as you go.) Thank you.

cc @antoniovicente

**Comment by akonradi** [comment]
Yeah I think #13800 is my last PR to add baked-in scaled timers. Just musing, I think things worth covering include:
- the different types of scaled timers
- how to construct scaled timers in custom filters
- warning about too many different scaled timeout values

As for edge best practices, I'm not entirely sure what those are (yet).
